Description: Fix various issues found in desktop menu entry file
 - Fix issues found by desktop-file-validate (deprecated Encoding key, Version
   key with incorrect value).
 - Change Icon path to a valid path on Debian systems.
 - Make translations less specific to localization. See #667640
 - Original .desktop file in past STK versions on Debian had extra fields in
   Categories, which ended up showing STK's menu entry in multiple locations;
   remove these extra fields. See LP: #709987
Forwarded: https://sourceforge.net/apps/trac/supertuxkart/ticket/632
Bug-Debian: http://bugs.debian.org/667640
Bug-Ubuntu: https://bugs.launchpad.net/bugs/709987
Last-Update: 05-10-2012

--- a/data/supertuxkart_desktop.template
+++ b/data/supertuxkart_desktop.template
@@ -1,16 +1,15 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=SuperTuxKart
-Version=VERSION
-Icon=PREFIX/share/pixmaps/supertuxkart_64.xpm
+Version=1.0
+Icon=supertuxkart
 GenericName=A kart racing game
-GenericName[de_DE]=Ein Kart-Rennspiel
-GenericName[fr_FR]=Un jeu de karting
+GenericName[de]=Ein Kart-Rennspiel
+GenericName[fr]=Un jeu de karting
 GenericName[gl]=Xogo de carreiras con karts
-GenericName[ro_RO]=Un joc de curse cu carturi
-Exec=PREFIX/games/supertuxkart --log=file
+GenericName[ro]=Un joc de curse cu carturi
+Exec=supertuxkart --log=file
 Terminal=false
 StartupNotify=false
 Type=Application
-TryExec=PREFIX/games/supertuxkart
+TryExec=supertuxkart
 Categories=Game;ArcadeGame;
\ No newline at end of file
